content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,122 @@
+| **Data Proxy**<br /><br /> - CPU usage        | 100,000<br /><br />CPU&nbsp;seconds<br /><br />(hard&nbsp;limit)               | 3,000<br /><br />CPU&nbsp;seconds<br /><br />(hard&nbsp;limit) | Unlimited<br /><br /><strong>Quota</strong><br /><br />30,000 CPU&nbsp;seconds<br /><br /><strong>Overage</strong><br /><br />Each additional CPU&nbsp;second at a fixed price | Unlimited<br /><br />Custom pricing                                            |
+| **Data Proxy**<br /><br /> -&nbsp;Concurrency | Low                                                                            | High                                                           | High                                                                                                                                                                           | Custom                                                                         |
+
+- To learn more about our pricing, see the [Pricing](https://prisma.io/pricing) page.
+- To upgrade to a paid plan, see [Upgrading and downgrading plans](/concepts/data-platform/billing/manage-plans).
+
+## Quotas
+
+### Quotas for team members and environments
+
+- You can add as many team members and environments as needed to a project.
+- We bill for each additional team member and environment that exceeds the quota at a fixed price based on the plan.
+- When you add a team member or environment that exceeds the quota, we first bill you with a pro-rated amount until the end of the current month. This is included in the invoice for next month.
+- After that, we bill you monthly at the fixed price per collaborator or environment for your plan.
+- If at any point during the current month you remove any previously added collaborators or environments from the monthly quota, we issue a refund for any unused time in the invoice for the next month.
+
+### Quotas for the Prisma Data Proxy
+
+- Each plan includes different Data Proxy CPU usage quotas which reset at the beginning of every month.
+- The **Early Adopter** and **Free** plans have a hard limit of monthly Data Proxy CPU usage.
+- If a project reaches the hard limit on **Early Adopter** and **Free** plans, we will pause your project until the beginning of next month when the Data Proxy CPU usage quota resets.
+- We will send notifications in advance before your project reaches the hard limit, so that you have time to consider upgrading to a paid plan to avoid service interruptions.
+- On the **Scale** and **Enterprise** plans, the Data Proxy usage quota is unlimited (no hard limits). Both plans include an initial quota of Data Proxy usage and additional usage exceeding the quota is billed on a monthly basis.
+- We do not issue refunds for unused Data Proxy CPU usage quotas.
+
+## <strong>Free</strong> plan
+
+The **Free** plan is suitable as a project playground for any project before it reaches the phase of scaling to a mature Prisma application that requires more resources to scale.
```

Comment:
My feeling is that it would be difficult to say that any project is small or smaller than another. At least, my understanding is that nothing prevents you from developing a huge project under **Free** and only after it starts having greater usage, it can be upgrade to a paid plan.

Reply:
I will not apply this for now.